MODULE.bazel

```diff
@@ -68,15 +68,20 @@ pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
 use_repo(pip, "pip_deps")
 
 crate = use_extension("@rules_rust//crate_universe:extension.bzl", "crate")
-crate.spec(
-    package = "googletest",
-    version = ">0.0.0",
-)
-crate.spec(
-    package = "paste",
-    version = ">=1",
+# As of rules_rust@0.52.2 crate.from_specs specifies the crate module extension
```

Comment:
nitL: I believe crate.from_specs *always* specified it as non-reproducible (I wrote it but my memory is fuzzy at times :) )

Reply:
Right sorry, I meant that, as of 0.52.2, it is "still" non-reproducible !